user_management_schema.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE `sessions` (
+    `session_id` varchar(128) NOT NULL,
+    `expires` int(11) unsigned NOT NULL,
+    `data` text,
+    PRIMARY KEY (`session_id`)
+  ) DEFAULT CHARSET=utf8;
+  
+  CREATE TABLE IF NOT EXISTS `users` (
+      `id` int(5) NOT NULL AUTO_INCREMENT,
+      `email_id` varchar(225) NOT NULL,
```

Comment:
We had made emai_id unique, didn't we? I guess the schema is not updated, kindly check.

Reply:
user doesn't seem to have username here, need to fix this schema.

Reply:
done.